### PR TITLE
Call job: ansible-test-sanity

### PR DESCRIPTION
**Pull request by gundalow**
Depends-On: https://github.com/ansible-network/ansible-zuul-jobs/pull/9

### CODE REVIEW

**Comment by gundalow** [comment]
recheck

**Comment by TristanCacqueray** [comment]
recheck

**Comment by TristanCacqueray** [comment]
recheck

**Comment by morucci** [comment]
recheck

**Comment by gundalow** [comment]
`tox.ini` added in https://github.com/ansible-network/network-engine-zuul/pull/7

recheck

**Comment by gundalow** [comment]
recheck

**Comment by gundalow** [comment]
recheck

**Comment by gundalow** [comment]
recheck

**Comment by gundalow** [comment]
Fixed `future` & `iteritems` in https://github.com/ansible-network/network-engine-zuul/pull/8

recheck

**Comment by gundalow** [comment]
recheck

**Comment by gundalow** [comment]
recheck

**Comment by gundalow** [comment]
recheck

**Comment by gundalow** [comment]
recheck

**Comment by gundalow** [comment]
recheck

**Comment by gundalow** [comment]
recheck

**Comment by gundalow** [comment]
https://github.com/ansible-network/network-engine-zuul/pull/11 has been merged which should fix the issues found by the sanity tests (invalid yaml & no return `type: dict`)

**Comment by gundalow** [comment]
recheck

**Comment by gundalow** [comment]
GREEEEEEEEEEEEEEEEEN

**Comment by gundalow** [comment]
recheck

**Comment by gundalow** [comment]
Ready for review:
Ara logs (click `CHANGED` for details) https://ansible.softwarefactory-project.io/logs/3/3/8e530e34d1efa8346640b1f52d74603d12404856/check/ansible-test-sanity/ba76f02/ara/
Or look at the raw logs at https://ansible.softwarefactory-project.io/logs/3/3/8e530e34d1efa8346640b1f52d74603d12404856/check/ansible-test-sanity/ba76f02/job-output.txt.gz#_2018-04-13_13_47_25_587484

File: .zuul.yaml [hunk 1]
```diff
@@ -3,3 +3,4 @@
       debug: True
```
Comment (TristanCacqueray) [inline .zuul.yaml]: maybe remove the debug now?

**Comment by gundalow** [comment]
recheck

**Comment by gundalow** [comment]
recheck

**Comment by gundalow** [comment]
recheck

**Pull request merged**